docs/apis/http/tus_upload.md

```diff
@@ -0,0 +1,178 @@
+< Tus-Resumable: 1.0.0
+< Vary: Origin
+< X-Content-Type-Options: nosniff
+< X-Download-Options: noopen
+< X-Frame-Options: SAMEORIGIN
+< X-Permitted-Cross-Domain-Policies: none
+< X-Request-Id: xxxxxxxxxxxxxxxxxxxxxx
+< X-Robots-Tag: none
+< X-Xss-Protection: 1; mode=block
+<
+* Connection #0 to host localhost left intact
+
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+The server will return a temporary upload URL in the location header of the response:
+```bash
+< Location: https://ocis.test/data/eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJhdWQiOiJyZXZhIiwiZXhwIjoxNjk3NTMyNTc5LCJpYXQiOjE2OTc0NDYxNzksInRhcmdldCI6Imh0dHA6Ly9sb2NhbGhvc3Q6OTE1OC9kYXRhL3R1cy8zYTU3ZWZlMS04MzE0LTQ4MGEtOWY5Ny04N2Q1YzBjYTJhMTgifQ.FbrlY7mdOfsbFgMrP8OtcHlCEq72a2ZVnPD2iBo9MfM
+```
+
+### Upload the First Chunk
+Once a temporary upload URL has been created, a client can send a PATCH request to upload a file. The file content should be sent in the body of the request:
+{{< tabs "upload-the-first-chunk" >}}
+{{< tab "Request" >}}
+```bash
+curl -ks -u admin:admin -XPATCH https://temporary-upload-url \
+-H "Tus-Resumable: 1.0.0" \
+-H "Upload-Offset: 0" \
+-H "Content-Type: application/offset+octet-stream" -d "01234"
+```
+{{< /tab >}}
+{{< tab "Response - 204 No Content" >}}
+```xmllint --format
+< HTTP/1.1 204 No Content
+< Date: Tue, 17 Oct 2023 04:10:52 GMT
+< Oc-Fileid: 8d72036d-14a5-490f-889e-414064156402$73bb5450-816b-4cae-90aa-1f96adc95bd4!84e319e4-de1d-4dd8-bbd0-e51d933cdbcd
+< Tus-Resumable: 1.0.0
+< Upload-Expires: 1697602157
+< Upload-Offset: 5
+< Vary: Origin
+< X-Content-Type-Options: nosniff
+< X-Request-Id: xxxxxxxxxxxxxxxxxxxxxx
+<
+* Connection #0 to host localhost left intact
+```
+{{< /tab >}}
+{{< /tabs >}}
+
+### Upload the Second Chunk
+After the first chunk is uploaded, the second chunk can be uploaded by pointing `Upload-Offset` to exact position that was returned in the first response
+{{< tabs "upload-the-second-chunk" >}}
+{{< tab "Request" >}}
+```angular2html
+curl -ks -u admin:admin -XPATCH https://temporary-upload-url \
+-H "Tus-Resumable: 1.0.0" \
+-H "Upload-Offset: 5" \
+-H "Content-Type: application/offset+octet-stream" -d "56789"
+```
+{{< /tab >}}
+{{< tab "Response - 204 No Content" >}}
+```xmllint --format
+< HTTP/1.1 204 No Content
+< Date: Tue, 17 Oct 2023 04:11:00 GMT
+< Oc-Fileid: 8d72036d-14a5-490f-889e-414064156402$73bb5450-816b-4cae-90aa-1f96adc95bd4!84e319e4-de1d-4dd8-bbd0-e51d933cdbcd
+< Tus-Resumable: 1.0.0
+< Upload-Expires: 1697602157
+< Upload-Offset: 10
+< Vary: Origin
+< X-Content-Type-Options: nosniff
+< X-Request-Id: xxxxxxxxxxxxxxxxxxxxxx
+<
+* Connection #0 to host localhost left intact
+```
+{{< /tab >}}
+{{< /tabs >}}
+{{< hint type=warning title="Important Warning" >}}
+`Upload-Offset` header indicates the byte position in the target file where the server should start writing the upload content.
+It ensures data integrity and order during the upload process.
+{{< /hint >}}
+
```

Comment:
maybe mention also that the upload will be finished when all bytes that were mentioned while creating the temporary URL are uploaded ( Header: `Upload-Length: 10"`)